_pages/dev/ssr/server-side-rendering-error-handling.md

```diff
@@ -279,13 +300,14 @@ export class MyActionFail implements ErrorAction {
   }
 }
 ```
-Please make sure that all your custom NgRx Actions of type Fail contain the property `error`.
 
-**Note:** If in your case some exceptional NgRx actions with the property `error` should not make SSR fail, you can customize the behavior by over-providing Spartacus `ErrorActionService`.
+Ensure that all your custom NgRx Actions of type `Fail` contain the `error` property.
+
+**Note:** If, in your case, some exceptional NgRx actions with the `error` property should not make SSR fail, you can customize the behavior by over-providing the Spartacus `ErrorActionService`.
 
-## Don't use the RESPONSE token for setting status code
+## Not Using the RESPONSE Token for Setting Status Codes
```

Comment:
The intention of this header was to warn the user what not to do. With the change made, when I read this header, the first that came to my mind was "ok, here is the paragraph about what happens if I don't use RESPONSE token for status codes" so it has the opposite effect 🤔 

I'm happy to hear your thoughts about this @Platonn @gladius-mtl.

Reply:
I understand what you are saying, but we can't put "Don't Use" in a heading... What if it said "Avoiding Use of the RESPONSE Token..." ?

Reply:
(the reason why we can't make a heading like that is that headings tell what a section is about, but they do not contain directives (actions, steps)...

Reply:
Understood! Thank you for the clarification. In that case, I vote for `Avoiding Use of the RESPONSE Token...`

Reply:
👍 🙇 🙂